tests/test_dask_to_root.py

```diff
@@ -0,0 +1,159 @@
+    dask_arr = dask_awkward.from_awkward(ak.from_iter(arr), partitions)
+    uproot.dask_write(
+        dask_arr, "/Users/zobil/Documents/uproot5/src/uproot/my-output", prefix="data"
+    )
+    file_1 = uproot.open(
+        "/Users/zobil/Documents/uproot5/src/uproot/my-output/data-part0.root"
+    )
+    assert len(file_1["tree"]["Jet_Px"].arrays()) == math.ceil(len(arr) / partitions)
+    file_2 = uproot.open(
+        "/Users/zobil/Documents/uproot5/src/uproot/my-output/data-part1.root"
+    )
+    assert len(file_2["tree"]["Jet_Px"].arrays()) == int(len(arr) / partitions)
+
+
+def zip_test():
+    """
+    Trying to find a way around duplicate counter issue.
+    """
+    partitions = 2
+    arr = uproot.open(skhep_testdata.data_path("uproot-HZZ.root"))["events"].arrays()
+    uproot.open(skhep_testdata.data_path("uproot-HZZ.root"))["events"].show()
+    zipped = ak.Array(
+        [
+            {
+                "Muon_": ak.zip(
+                    {
+                        name[5:]: array
+                        for name, array in zip(ak.fields(arr), ak.unzip(arr))
+                        if name.startswith("Muon_")
+                    }
+                )
+            },
+            {
+                "Jet_": ak.zip(
+                    {
+                        name[: name.index("_")]: array
+                        for name, array in zip(ak.fields(arr), ak.unzip(arr))
+                        if name.startswith("Jet_")
+                    }
+                )
+            },
+            {
+                "Photon_": ak.zip(
+                    {
+                        name[: name.index("_")]: array
+                        for name, array in zip(ak.fields(arr), ak.unzip(arr))
+                        if name.startswith("Photon_")
+                    }
+                )
+            },
+            {
+                "Electron_": ak.zip(
+                    {
+                        name[: name.index("_")]: array
+                        for name, array in zip(ak.fields(arr), ak.unzip(arr))
+                        if name.startswith("Electron_")
+                    }
+                )
+            },
+        ]
+    )
+    dask_arr = dask_awkward.from_awkward(ak.from_iter(zipped), partitions)
+    uproot.dask_write(
+        dask_arr, "/Users/zobil/Documents/uproot5/src/uproot/my-output", prefix="data"
+    )
+    file_1 = uproot.open(
+        "/Users/zobil/Documents/uproot5/src/uproot/my-output/data-part0.root"
+    )
+    assert len(file_1["tree"]["Jet_Px"].arrays()) == math.ceil(len(arr) / partitions)
+    file_2 = uproot.open(
+        "/Users/zobil/Documents/uproot5/src/uproot/my-output/data-part1.root"
+    )
+    assert len(file_2["tree"]["Jet_Px"].arrays()) == int(len(arr) / partitions)
+
+
+def HZZ_test_3_partitions():
+    """
+    Write data from HZZ with 3 partitions
+    """
+    partitions = 3
+    arr = uproot.open(skhep_testdata.data_path("uproot-HZZ.root"))["events"].arrays()
```

Comment:
At least one of your tests should use a distributed.Client like:
```python3
from distributed import Client

def test_something():
    with Client() as _:
        #do dask things
```

This will ensure everything you've written is serializable and shippable to worker nodes on a cluster!